website/src/releases/2021-04/release-v19.mdx

```diff
@@ -46,14 +46,14 @@ The equivalent mapping from `constraint` to `max` is as following:
 
 ## Text
 
-For the `<Text.Headline>` and `<Text.Subheadline>` components, the deprecated `elementType` prop has been removed. Now only the `as` prop is used instead.
+For the `<Headline>` and `<Subheadline>` components, the deprecated `elementType` prop has been removed. Now only the `as` prop is used instead.
```

Comment:
I was wondering why we are changing these components names since it seems to me the full qualified name is more clear.
Is it because the new MDX version does not support them outside code blocks?

Reply:
It's related to the Vale style checker (the writing-style package). We managed to fix these issues in the code blocks but we still have this issue with the inline code.

https://github.com/commercetools/commercetools-docs-kit/issues/1341

It's a long standing issue with no clear fix unfortunately.

Reply:
Ah! Thanks for the explanation.

I was thinking though, that perhaps we could use the meta-comments suggested in that issue to be able to use the qualified name since I believe our users would better understand where those components come from, but I'm not sure is those comments are only valid for code blocks.

![image](https://user-images.githubusercontent.com/97907068/218413367-71595ee2-996c-472b-9c15-f13530c94929.png)

Reply:
I managed to make it work (disabling vale for that specific line) by wrapping the vale comment into the mdx v2 comment 😅

https://github.com/commercetools/merchant-center-application-kit/pull/2961/commits/8caf15a9504a13371b27a8fbc9adc0e78836fe27

Thanks for double checking!